<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsoStufferFrame</class>
 <widget class="QMainWindow" name="IsoStufferFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Isosurface Stuffing</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/fracture.png</normaloff>:/images/fracture.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="IsoStufferCanvas" name="canvas" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportStellar"/>
    </widget>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionCenterize"/>
    <addaction name="actionFlipTgl"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionSaveMesh"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionWireframe"/>
    <addaction name="actionFacet"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>&amp;Process</string>
    </property>
    <addaction name="actionStep"/>
    <addaction name="actionConvert"/>
    <addaction name="separator"/>
    <addaction name="actionParams"/>
    <addaction name="separator"/>
    <addaction name="actionCheckMesh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionLoad"/>
   <addaction name="actionReload"/>
   <addaction name="separator"/>
   <addaction name="actionStep"/>
   <addaction name="actionConvert"/>
   <addaction name="separator"/>
   <addaction name="actionWireframe"/>
  </widget>
  <action name="actionLoadBody">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/load_frac.png</normaloff>:/images/load_frac.png</iconset>
   </property>
   <property name="text">
    <string>Load Rigid Body</string>
   </property>
  </action>
  <action name="actionExportMatlab">
   <property name="text">
    <string>&amp;Export Matlab Data</string>
   </property>
  </action>
  <action name="actionExportAbaqus">
   <property name="text">
    <string>&amp;Export Abaqus Mesh</string>
   </property>
  </action>
  <action name="actionModeShapes">
   <property name="text">
    <string>Mode Shapes</string>
   </property>
  </action>
  <action name="actionLoadModes">
   <property name="text">
    <string>Load Modes</string>
   </property>
  </action>
  <action name="actionExportMatrices">
   <property name="text">
    <string>Export Matrices</string>
   </property>
  </action>
  <action name="actionExportSurfaceMesh">
   <property name="text">
    <string>Export Surface Mesh</string>
   </property>
  </action>
  <action name="actionExportImpulses">
   <property name="text">
    <string>Export Impulses</string>
   </property>
  </action>
  <action name="actionFacet">
   <property name="text">
    <string>&amp;Facet</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/grid.png</normaloff>:/images/grid.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Wire frame</string>
   </property>
  </action>
  <action name="actionParams">
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/load.png</normaloff>:/images/load.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionCenterize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Centerize</string>
   </property>
  </action>
  <action name="actionFlipTgl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flip triangles</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/load_frac.png</normaloff>:/images/load_frac.png</iconset>
   </property>
   <property name="text">
    <string>Reload current mesh</string>
   </property>
  </action>
  <action name="actionSaveMesh">
   <property name="text">
    <string>Save tetra mesh</string>
   </property>
  </action>
  <action name="actionExportStellar">
   <property name="text">
    <string>TetGen/Stellar Format</string>
   </property>
  </action>
  <action name="actionCheckMesh">
   <property name="text">
    <string>Check mesh</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IsoStufferCanvas</class>
   <extends></extends>
   <header>ui/IsoStufferCanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>IsoStufferFrame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
